src/cryptoadvance/specter/services/static/CustomElement.js

```diff
@@ -0,0 +1,97 @@
+
+export class CustomElement extends HTMLElement {
```

Comment:
@k9ert this doesn't seem to be used anywhere. Should this be removed from this PR?